API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
```

Comment:
I would say optional. We've been following this convention internally, but I wouldn't expect others in the network to do the same.